dags/public-health/covid19/jhu-to-esri.py

```diff
@@ -156,26 +116,66 @@ def load_jhu_global_time_series(branch="master"):
 
     # join
     df = df.assign(
-        number_of_deaths=deaths_df.deaths, number_of_recovered=recovered_df.recovered,
+        number_of_deaths=deaths_df.deaths,
+        number_of_recovered=recovered_df.recovered,
+        date=pd.Timestamp.now(tz="US/Pacific").normalize().tz_convert("UTC"),
     )
 
     return df.sort_values(["date", "Country/Region", "Province/State"]).reset_index(
         drop=True
     )
 
 
-def load_esri_time_series(gis):
+def load_jhu_global_current(**kwargs):
     """
-    Load the county-level time series dataframe from the ESRI feature server.
+    Loads the JHU global current data, transforms it so we are happy with it.
     """
-    gis_item = gis.content.get(time_series_featureid)
-    layer = gis_item.layers[0]
-    sdf = arcgis.features.GeoAccessor.from_layer(layer)
+    # Login to ArcGIS
+    arcconnection = BaseHook.get_connection("arcgis")
+    arcuser = arcconnection.login
+    arcpassword = arcconnection.password
+    gis = GIS("http://lahub.maps.arcgis.com", username=arcuser, password=arcpassword)
+
+    # (1) Load current data from ESRI
+    gis_item = gis.content.get(JHU_GLOBAL_SOURCE_ID)
+    cases_layer = gis_item.layers[1]
+    sdf = arcgis.features.GeoAccessor.from_layer(cases_layer)
+    # ESRI dataframes seem to lose their localization.
+    cases = sdf.assign(date=sdf.date.dt.tz_localize("UTC"))
     # Drop some ESRI faf
-    sdf = sdf.drop(columns=["ObjectId", "SHAPE"]).drop_duplicates(
-        subset=["date", "county"], keep="last"
+    cases = cases.drop(columns=["ObjectId", "SHAPE"])
+
+    deaths_layer = gis_item.layers[0]
+    sdf = arcgis.features.GeoAccessor.from_layer(deaths_layer)
+    # ESRI dataframes seem to lose their localization.
+    deaths = sdf.assign(date=sdf.date.dt.tz_localize("UTC"))
+    # Drop some ESRI faf
+    deaths = deaths.drop(columns=["ObjectId", "SHAPE"])
+
+    # melt cases
+    id_vars, dates = parse_columns(cases)
+    df = pd.melt(
```

Comment:
Do we need to `melt` here? I thought that was only relevant for the CSV from GitHub.